packages/server/lib/utils/response-error.ts

```diff
@@ -0,0 +1,24 @@
+import type { Response } from 'express';
```

Comment:
🍖 new helper function wrapping up sending a 500 with a trace and a log. We technically already log the body on the 500, but it happens after the response which I'm pretty sure is what is making it harder to trace, and I don't love that we output the error so far from where it happens.